charts/tezos/templates/nodes.yaml

```
@@ -66,10 +66,16 @@ spec:
 {{- if $.node_vals.local_storage | default false }}
         - emptyDir: {}
           name: var-volume
-{{- else }}
+{{- end }}
   volumeClaimTemplates:
     - metadata:
+{{- if $.node_vals.local_storage | default false }}
+        # since we are using local storage, deploy a small persistent voluem
+        # to store identity.json and peers.json
```

```suggestion
        # since we are using local storage, deploy a small persistent volume
        # to store identity.json and peers.json
```
Also it may need to be wrapped in {{ /* */}}. Unless helm template works

---

helm template works